core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -6253,13 +6253,17 @@ components:
     StateConsensusManagerResponse:
       type: object
       required:
+        - state_version
```

Comment:
For consistency / clarity, I'd suggest we put this in some `ledger_version` wrapper object (or `at_ledger_state` [like the Gateway API](https://radix-rcnet-v2-gateway.redoc.ly/#operation/StateEntityDetails)) which contains the state version and ledger hash stuff - that way, it's a little more separate from the rest of the response.

I'd even be up for if you'd like to put the rest of the response in some `content` or `value` or `data` wrapper, for clarity.

Reply:
> For consistency / clarity, I'd suggest we put this in some `ledger_version` wrapper object (or `at_ledger_state` ...

I am all in - and actually then it would make sense to pull the `state_version` into the `LedgerHeaderSummary(ledger_hashes, proposer_ts)`, as described at https://github.com/radixdlt/babylon-node/pull/586#issue-1837218388. The only awkward question left would be "how do we shape it in the existing LTS endpoints?".

> I'd even be up for if you'd like to put the rest of the response in some content or value or data wrapper

If we are adding the dedicated `at_ledger_state`/`ledger_header_summary`, then wrapping the rest too seems like an overkill.